sql/moz-fx-data-marketing-prod/ga_derived/blogs_daily_summary_v1/query.sql

```diff
@@ -0,0 +1,44 @@
+SELECT
+  date,
+  device_category,
+  operating_system,
+  browser,
+  `language`,
+  country,
+  standardized_country_list.standardized_country AS standardized_country_name,
+  source,
+  medium,
+  campaign,
+  content,
+  blog,
+  subblog,
+  SUM(sessions) AS sessions,
+  SUM(downloads) AS downloads,
+  SUM(social_share) AS social_share,
+  SUM(newsletter_subscription) AS newsletter_subscription,
+FROM
+  `moz-fx-data-marketing-prod.ga_derived.blogs_sessions_v1` AS sessions_table
```

Comment:
```suggestion
  blogs_sessions_v1 AS sessions_table
```

Per https://github.com/mozilla/bigquery-etl#queries our preferred style is to make table references as short as possible, so we should consistently leave off the project and dataset where they match the target project and dataset.